src/cobald/daemon/runners/meta_runner.py

```diff
@@ -34,7 +34,8 @@ def runners(self):
             DeprecationWarning(
                 "Accessing 'MetaRunner.runners' directly is deprecated. "
                 "Use register_payload or run_payload with the correct flavour instead."
-            )
+            ),
+            stacklevel=2,
```

Comment:
Fixed according to flake8-bugbear:

> **B028:** No explicit stacklevel keyword argument found. The warn method from the warnings module uses a stacklevel of 1 by default. This will only show a stack trace for the line on which the warn method is called. It is therefore recommended to use a stacklevel of 2 or greater to provide more information to the user.